modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/allow_list_sync_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class AllowListSyncJob
+    include Sidekiq::Job
+    sidekiq_options retry: false
+
+    # To not overload the DB.
+    MAX_RECORD_COUNT = 500
+
+    Error = Class.new(RuntimeError)
+
+    class RecordCountError < Error
+      def initialize(record_count)
+        super("record_count: #{record_count}")
+      end
+    end
+
+    def perform
+      SemanticLogger.tagged do
+        csv = extract
+        csv.size.between?(1, MAX_RECORD_COUNT) or
+          raise RecordCountError, csv.size
+
+        attributes = transform!(csv)
+        result = load(attributes)
+
+        logger.info(result)
+      rescue => e
+        logger.error(e)
+        raise
+      end
+    end
+
+    private
+
+    def extract
+      config = Settings.accredited_representative_portal.allow_list.github
+      Octokit::Client
+        .new(access_token: config.access_token)
```

Comment:
How often do you thing this job will run? I'm wondering if this could get rate limited if it's called too frequently.

Reply:
We haven't installed it in `periodic_jobs.rb` yet, but my thought was once every 15 minutes. Would this likely be rate limited?